pandas/core/generic.py

```diff
@@ -7623,10 +7624,8 @@ def between_time(
             Initial time as a time filter limit.
         end_time : datetime.time or str
             End time as a time filter limit.
-        include_start : bool, default True
-            Whether the start time needs to be included in the result.
-        include_end : bool, default True
-            Whether the end time needs to be included in the result.
```

Comment:
these should still be documented, just add a `deprecated` directive (see some other docstrings for examples)

Reply:
Yes, doing both :)

Reply:
@MarcoGorelli I've made commits as per your last review. Please take a look and let me know if anything else is required :)

Reply:
Edit: I'm not sure what exactly are you referring to as a `deprecated` directive in a docstring. Can you please give me an example?

Reply:
What I did was to edit the whatsnew file

Reply:
Previously, I went ahead and edited the `whatsnew` file.

Reply:
@MarcoGorelli Can you please make a suggestion as to how I should put `deprecated` in the docstring? Is this correct?

```
        start_time : datetime.time or str
            Initial time as a time filter limit.
        end_time : datetime.time or str
            End time as a time filter limit.
        axis : {0 or 'index', 1 or 'columns'}, default 0
            Determine range time on index or columns value.
        include_start : bool, default True .. deprecated:: 0.21.0
            Whether the start time needs to be included in the result.
        include_end : bool, default True .. deprecated:: 0.21.0
            Whether the end time needs to be included in the result.
        inclusive: {"both", "neither", "left", "right"}, default "both"
            Include boundaries; whether to set each bound as closed or open.
```

Reply:
you need the spaces: looks something like this:

```
arg : bool, optional
    does something

    .. depreacted:: 1.4.0
       Some message about why its deprecated, use other arg, etc.
```

Reply:
Thank you! understood :)